faker/providers/internet/__init__.py

```diff
@@ -88,6 +89,39 @@ def company_email(self):
     def free_email_domain(self):
         return self.random_element(self.free_email_domains)
 
+    def uuid4_email(self):
```

Comment:
I don't really see the use case for uuid4 emails. The goal of Faker is provide plausible values, and I don't think there are many people using uuid4 for their email.

Reply:
@fcurella:

Well, I've added that, because I needed it. I use faker as a generator of test data. My last use case was to generate 100 POST requests to an REST API. Emails had to be unique. I have solved it by now by registering a custom provider, but purely for testing, absolutely unique emails are handy.

Reply:
I understand your use case, but I don't think it's common enough to be included as a convenience.
For people who needs an uuid4 email, It's still possible to generate one by simply calling `faker.uuid4() + '@' + faker.domain_name()`